backend/src/modules/Comment/CommentModule.js

```diff
@@ -1,5 +1,5 @@
 import { CommentController } from './controller/Comment.controller.js';
```

Comment:
CommentModule.js eksportuje funkcję, to raczej chcemy: nazwę eksportowanej funkcji  małą literą.
`export const commentModule = (moduleRepositoryType) => {`